### DISCUSSION TITLE
Recommended routes for CLI & API web sub-sections?

### DISCUSSION THREAD

**Discussion by ilg-ul** [facebook/docusaurus#10594] [Q&A] [answered]
I have a TS project that, in addition to the usual API, also provides a CLI.

The documentation pages are visible below `docs`, and blog posts below `blog`, as usual.

However, I'm not sure where the `cli` and `api` should be located. My first thought was to place them at the top. Is this a reasonable solution? Or should I place them below `docs`?

I know that placing them at the top requires multiple instances for the `plugin-content-docs` plugin, but that is not a problem, I already have functional configurations with multiple docs and blog plugins.

@slorber, if you'd have such a project, where would you place the `cli` and `api` web sub-sections, at the top or inside `docs`?

Thank you for your valuable help!

**Comment by slorber** [comment, marked as answer]
Our own website has an API section: https://docusaurus.io/docs/cli

Isn't this setup good for your use-case? We only use 1 docs instance, but have split it into 2 distinct sidebars.

The problem with multi-instance + versioning is that it makes it harder to link from one instance to another. We only support linking this way through `/pathname` and not `./filename.mdx`

**Reply by ilg-ul** [reply]
Ok, I'll do the same, I'll define both CLI and API below `docs`, using a single instance.

Thank you!
